wls-gui-admintool/src/composables/wahltermindaten/wahltermindatenService.ts

```diff
@@ -0,0 +1,58 @@
+import { ref } from "vue";
+
+import {
+  Configuration,
+  WahltermindatenControllerApi,
+} from "@/api/wls-clients/generated-admin-api";
+import { useUserNotificationService } from "@/composables/userNotification/userNotificationService.ts";
+import { ADMIN_SERVICE_API_URL } from "@/constants.ts";
+
+const controllerApi = new WahltermindatenControllerApi(
+  new Configuration({
+    basePath: ADMIN_SERVICE_API_URL,
+  })
+);
+const { addNotification } = useUserNotificationService();
+
+export function useWahltermindatenService() {
+  const isLoading = ref(false);
+  const isDeleting = ref(false);
+
+  async function importWahlterminDaten(wahltagID: string) {
+    isLoading.value = true;
+    try {
+      await controllerApi.loadWahltermindaten(wahltagID);
+    } catch {
+      addNotification(
+        "Importieren der Wahltermindaten fehlgeschlagen",
+        "Error"
+      );
+    }
+    isLoading.value = false;
```

Comment:
```suggestion
    } finally {
      isLoading.value = false;
    }  
```

warum unten mit finally und hier nicht?

Reply:
musste ich im Rahmen von d87391346b27c4ea49bded00964c5f1472b35fc7 anpassen. Ist jetzt in beiden fällen im Finally Block